omod/src/main/java/org/openmrs/module/patientqueueapp/page/controller/PacRoomQueuePageController.java

```diff
@@ -18,7 +18,7 @@
  */
 @AppPage(PatientQueueConstants.APP_PAC)
 public class PacRoomQueuePageController {
-    private static final String PAC_ROOM_CONCEPT_UUID = "164820AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+    private static final String PAC_ROOM_CONCEPT_UUID = "57287737-988c-4fd1-a3a2-0e9d9a7dd15d";
```

Comment:
did you get this already in the concept dictionary or you created yourself?

Reply:
this was preexisting ID: 165445

Reply:
so this points to family planning, how is this related to PAC, if we use a concept that can expalin what we are doing could be great, what happened to the first one

Reply:
PAC is part of family planning so to avoid creating new concept settled with FP.

Reply:
OK